src/Printer.java

```diff
@@ -0,0 +1,8 @@
+public class Printer {
+    int a = 45;
+    public int printParams (){
+        String formatingData = String.format("\"---<%d>---\"", a);
+        System.out.println(formatingData);
```

Comment:
This could be simplified to `System.out.printf("\"---<%d>---\"%n", a);`, just suggestion